pandas/tests/arrays/categorical/test_analytics.py

```diff
@@ -5,14 +5,13 @@
 
 from pandas.compat import PYPY
 
-from pandas import Categorical, Index, Series
+from pandas import Categorical, Index, Series, date_range
 from pandas.api.types import is_scalar
 import pandas.util.testing as tm
 
 
 class TestCategoricalAnalytics:
-    def test_min_max(self):
-
+    def test_min_max_not_ordered(self):
```

Comment:
```suggestion
    def test_min_max_not_ordered_raises(self):
```

Thanks for splitting this test, if you can also parametrize with min, max that'll be great.

Reply:
Great suggestion, on it!